x-pack/legacy/plugins/siem/public/components/embeddables/actions/apply_siem_filter_action.test.tsx

```diff
@@ -98,20 +109,22 @@ describe('ApplySiemFilterAction', () => {
 
   describe('#execute', () => {
     test('it throws an error when triggerContext not set', async () => {
-      const action = new ApplySiemFilterAction({ applyFilterQueryFromKueryExpression });
+      const action = new ApplySiemFilterAction({
+        applyFilterQueryFromKueryExpression,
+      });
       const embeddable = {
         type: MAP_SAVED_OBJECT_TYPE,
       };
-      if (isEmbeddable(embeddable)) {
-        const error = expectError(() =>
-          action.execute({
-            embeddable,
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          } as any)
-        );
-        expect(error).toBeInstanceOf(Error);
+      if (isPartialFilterAction(action) && isEmbeddable(embeddable)) {
+        if (isEmbeddable(embeddable)) {
+          await expect(
+            action.execute({
+              embeddable,
+            })
+          ).rejects.toThrow('Applying a filter requires a filter as context');
```

Comment:
Thanks for making this check a bit more strict -- don't want a different error being thrown to get this test to pass! 🙂